---
layout: default
title: WEB LOG
---
<h1>Yo, click the play button ^ </h1>
Translation of the mRNA-encoded genetic information into proteins is catalyzed by the intricate ribonucleoprotein machine, the ribosome. Historically, the bacterial ribosome is viewed as an unchangeable entity, constantly equipped with the entire complement of RNAs and proteins. Conversely, several lines of evidence indicate the presence of functional selective ribosomal subpopulations that exhibit variations in the RNA or the protein components and modulate the translational program in response to environmental changes. Here, we summarize these findings, which raise the functional status of the ribosome from a protein synthesis machinery only to a regulatory hub that integrates environmental cues in the process of protein synthesis, thereby adding an additional level of complexity to the regulation of gene expression.
<p>
  Introduction
The bacterial ribosome is a large ribonucleoprotein particle with an approximate molecular mass of 2.5 MDa and a diameter of 200–250 A ̊ . This intricate macromolecular machine is conducting the functionally complex translation of the mRNA-based genetic infor- mation into the amino acid sequence of proteins. This process is conceptually divided into four phases: first, initiation, where the ribosome — equipped with initiator tRNA — is positioned at the translation initiation region of the mRNA; second, elongation, when polypeptide synthesis occurs; and third, termination, where the newly synthesized protein is released after the ribosome encounters a stop codon; these steps are followed by; fourth, recycling of the ribosome into its subunits for the next round of protein synthesis. The underlying struc- tural complexity of the ribosome became apparent by the work of Waller in 1964, who demonstrated the presence of
a vast number of different ribosomal proteins (r-proteins) [1]. This finding initiated a long-term debate on the homogeneity of bacterial ribosomes [2,3], which was challenged by several studies performed in the early 1970s indicating the existence of ribosomal subpopu- lations that vary in their protein complement in response to different growth rates and environmental conditions [4–8]. Despite these intriguing observations, bacterial ribosomes are traditionally still viewed as homogeneous entities that have to be equipped with the same comp- lement of r-proteins and rRNA molecules to precisely accomplish all steps in protein synthesis. This concept was further perpetuated and strengthened by the deter- mination of atomic resolution structures of the ribosome at the beginning of the millennium [9–12]. Consequently, ribosomes were not considered to have an intrinsic regu- latory capacity, and the efficiency of translation was suggested to be determined either by features inherent to the mRNA or mediated by protein or RNA regulators.
In contrast to this perception recently emerging evidence reinforced the notion that subpopulations of hetero- geneous and functionally specialized ribosomes are engendered when bacteria encounter environmental stress. In this review we summarize these discoveries and discuss several principles of ribosome heterogeneity, which underline the intrinsic capacity of ribosomes to act as key regulators of translation rather than just as protein synthesis machineries. Besides the ribosome diversifica- tion induced by external cues, we likewise discuss the concept of intrinsic ribosome heterogeneity conveyed by variations of the modification of ribosomal RNA and protein components (Box 1; Figure 1). By virtue of these alterations, which could potentially also be regulated, the bacterial cell might be equipped with functional diverse translational machineries even under relaxed conditions.
General features of the ribosome
The bacterial ribosome (70S) is composed of two asym- metric subunits, the 30S and the 50S subunit, which assemble at the ribosome binding site on the mRNA during translation initiation. Each subunit contributes to specific functions in protein synthesis. In Escherichia coli the small 30S subunit is composed of the 16S rRNA consisting of 1542 nucleotides (nts), and 21 proteins. Its shape is largely determined by the RNA component, which forms the three domains, the head, the platform, and the body (Figure 2a) [11]. It mediates the step of initiation and contains the binding sites for the three initiation factors as well as the messenger decoding
center, where the respective codons of the mRNA are base-paired with the anticodon of the cognate tRNA. The large 50S subunit consists of two different rRNAs, the 23S rRNA (2904 nts) and the 5S rRNA (120 nts), and 34 proteins. It catalyzes peptide bond formation at the peptidyl transferase center, provides the binding sites for the elongation factors and comprises the exit tunnel for the nascent peptide chain.
Variation of the ribosomal protein complement
In light of this structural and functional complexity a variety of different mechanisms leading to heterogeneity
and consequently to the generation of functionally specialized ribosomes can be envisaged. As already suggested by studies mentioned above, ribosomes could differ in the stoichiometry of r-proteins [4–8]. Despite the catalytic activity of the rRNA in protein synthesis, the variation of or deficiency in some r-proteins could con- tribute to a fine tuning of ribosome function and in particular to its selectivity for distinct transcripts, a mech- anism that is, well established in eukaryotes: a study performed already in 1981 in the slime mold Dictiostelium discoideum revealed that ribosomes from vegetative amoe- boid cells differ in their protein content from ribosomes derived from spores [13]. This idea was further put forward by Mauro and Edelman in their ‘ribosome filter hypothesis’ in which the ribosome acts like a filter that selects for specific mRNAs and hence modulates trans- lation [14]. Later on, the existence of ribosome protein paralogs in Saccharomyces cerevisiae, which contribute to the generation of functionally distinct ribosome popu- lations, supported the new level of complexity in the regulation of gene expression and the term ‘ribosome code’ was coined [15]. Conversely, in prokaryotes the ribosome was generally not considered as a source for translational regulation, despite the first indications of heterogeneity due to alteration in the stoichiometry of r- proteins made almost 40 years ago [3,5,16,17]. In 1972, Deusser and Wittmann described a growth-rate-depend- ent variation in the protein composition of E. coli ribo- somes. Their data revealed that the amount of proteins S6, S21, and L12 differs significantly, when ribosomes were prepared from cells grown either in rich or minimal media [5,7]. These observations are consistent with results published by Weber in 1972 [18] and Milne et al. in 1975 [8]. Moreover, protein S1, which is essential for translation initiation on canonical mRNAs [19], was found to be present in substoichiometric amounts on 70S ribosomes resulting in functional ribosome heterogeneity [16,20]. Surprisingly, since these early times in ribosome research only little attention has been dedicated to the regulatory role of the ribosome in bacterial translation.
More recently, several lines of evidence for the functional specificity of ribosomes harboring a reduced protein complement emerged from studies performed to enlighten the requirements for translation of leaderless mRNAs (lmRNAs), which harbor a 50-terminal AUG start codon and are thus devoid of canonical ribosome recog- nition signals [21–26]. Collectively, these studies suggest a model wherein a subpopulation of ribosomes lacking protein S1 is responsible for translation of leaderless transcripts in vivo [24]. This notion was further under- lined by a study performed to decipher the phenomenon of enhanced translational efficiency of lmRNAs in an E. coli rpsBts mutant strain conditionally deficient for r- protein S2 [21]. The results reveal that S2 is required for binding of protein S1 to the ribosome. Consequently, the lack of S1 can readily explain the inhibition of bulk
(canonical) mRNA translation in the E. coli rpsBts mutant strain under non-permissive conditions [21]. Taken together with the observation that S1 is present in sub- stoichiometric amounts on 70S ribosomes [20], these results implicated for the first time that ribosome hetero- geneity might contribute to the functional selectivity of the translational machinery under distinct conditions. Recently, this notion gained further support by additional data substantiating the presence of a subpopulation of S1- depleted ribosomes under normal physiological con- ditions in E. coli cells [27]. Thus, it is tempting to speculate that conditions that might increase the S1- deficient ribosomal subpopulation could specifically stimulate the translation of lmRNAs.
In line with these observations, a follow-up study per- formed to explore the selective effect of the aminoglyco- side antibiotic kasugamycin on translation initiation of
lmRNA revealed the appearance of ribosomes in E. coli that lack several r-proteins [28]. These ribosomes are depleted for at least six proteins from the 30S subunit, among which are the functionally important proteins S1 and S2. However, the characterization of these protein- depleted 61S particles revealed that they are competent in translation of lmRNAs even in the presence of kasu- gamycin in vivo and in vitro [28]. Notwithstanding the artificial conditions, which trigger the release of proteins S1, S2, S6, S18, S12, and S21 from the mature 70S ribosome (Figure 1), this study presented the first evi- dence for the functionality of ribosomes devoid of multiple proteins, and puts forward the idea that the protein complement of the ribosome can vary in response to changed physiological conditions, which in turn leads to a modulation of the activity and specificity of the protein synthesis machinery. However, it is important to note that in contrast to the formation of 61S particles,
where the r-proteins are removed from mature 70S ribo- somes in the presence of kasugamycin, it still remains elusive whether the observed generation of protein- depleted ribosomes [3–8,16–18,20] can be attributed to the incorporation of premature subunits or whether these r-proteins are removed or lost from mature ribosomes.
Variation of the ribosomal RNA complement
Considering the catalytic activity of the rRNA in ribo- some function, it is conceivable that variability within the RNA component could likewise represent a potential mechanism to establish functional ribosome heterogen- eity. The presence of multiple individual rrn operons in bacteria and the observed differential regulation of the seven rRNA operons present in E. coli, which slightly differ in sequence, add significant weight to this argu- ment [29–31] (Figure 1). Taken together these data support a model wherein sequence micro-heterogeneities of the rrn operons might contribute to the modulation of activity and specificity of the ribosome in response to environmental changes [29]. This hypothesis can be exemplified by the observation that during the morpho- logical development of the Gram-positive bacterium Streptomyces coelicolor the heterogeneous rRNA molecules encoded by distinct rRNA operons are differentially expressed and assembled into the ribosomes [32,33]. Considering that the differentiation of Streptomycetes generally coincides with the production of antibiotics, some of which target the protein synthesis machinery, these results underscore the possibility that variations in the rRNA may play a significant role in the control of gene expression in response to developmental cues [32]. How- ever, it is still an open question whether an alteration in translational specificity can be attributed to ribosomes containing different rRNA species.
The surprising discovery that the 16S rRNA of assembled ribosomes becomes 30-terminally truncated when E. coli cells encounter stress completely changed the perception of ribosome-mediated regulation of protein synthesis [34􏰀􏰀]. Stress-induced activation of the endoribonuclease MazF, the toxin component of the toxin-antitoxin system mazEF [35], results in cleavage of the 16S rRNA at the ACA-site located at positions 1500–1502 (Figure 3) [34􏰀􏰀]. Since thereby the anti-Shine Dalgarno sequence, which is crucial for translation initiation on canonical mRNAs, is removed, the altered ribosomes selectively translate lmRNAs that are likewise generated by MazF [34􏰀􏰀]. It is intriguing to note that this specialization of the transla- tional machinery does neither rely on de novo ribosome biogenesis nor on the synthesis of regulatory RNA or protein factors, since the MazF-mediated cleavage occurs on mature 30S subunits or 70S ribosomes. Hence, this ribosome modification represents a paradigm for a highly economical and fast-reacting regulatory mechanism, which is conceptually related to the translational regula- tion by riboswitches or ribozymes [36]. Moreover, as the
variation is achieved by virtue of RNA cleavage, which is independent of cell growth, this post-transcriptional stress response pathway might be of physiological relevance for cells encountering hostile conditions, e.g. during host infection. Thus, it is tempting to speculate that the shift of ribosome specificity during bacteriostasis provides a means to prepare the translational program of stressed cells for recuperation under ameliorated con- ditions, which would, for example, facilitate growth recovery of bacterial pathogens when the antibiotic treat- ment is stopped.
Conclusion and future directions
The results summarized in this review emphasize that the bacterial ribosome can effectively and economically inte- grate environmental signals in the regulatory network of protein synthesis. Since the ‘hot spots’ of ribosome varia- bility are mainly clustered at the 30S subunit, in close proximity to the decoding center (Figure 2), these studies collectively indicate that ribosome heterogeneity can be considered as a potential means to fine tune translation in response to environmental cues at the level of initiation, the major rate limiting step in protein synthesis [37,38]. Surprisingly, this regulatory role of the bacterial ribosome was not appreciated until recently, what might be attrib- uted at least partially to the main focus on the structural and functional characterization of the translational appar- atus, which was intentionally based on the purification of homogeneous ribosome populations from exponentially growing cells. In light of this consideration, the identifi- cation of more variations can be anticipated when study- ing the composition of ribosomes purified from bacteria that encountered adverse conditions. Moreover, the vast amount of data that becomes available by global RNA-seq analyses performed under diverse conditions could like- wise indicate the differential regulation of ribosomal components and thus potentially facilitate the search for heterogeneous ribosome populations.
In light of the enormous amount of energy required for ribosome biogenesis, one puzzling question raised by these findings, namely the fate of the specialized ribo- somes, still remains elusive. A single study addressing the mechanism of ribosome repair provided evidence that the replacement of damaged r-proteins restores translational activity of chemically inactivated ribosomes [39]. Since the majority of the proteins involved in ribosome hetero- geneity — with the exception of proteins S6 and S18 — are determined to be exchangeable, this study strongly corroborates the regulatory function of the ribosome and provides a possible mechanism to reverse the specializ- ation of the translational machinery. Therefore, the unex- pected observation that the specificity of fully assembled ribosomes, which are already translationally active, can be modulated has opened up new avenues for the bacterial stress response, as the ribosome represents a novel hub in the complex regulatory network of cell physiology.
